SU-SWS/decanter
# D8CORE-2630: fixup! on the specifity

# READY FOR REVIEW

# Summary
- Fixing the hovers for gold and grey alert boxes

# Needed By (Date)
- Sept 4

# Urgency
- Hight

# Steps to Test

1. Pull in this change.
2. Point your local site to this decanter change 
3. Verify the Gold and Grey box has no color change on hover.

# Affected Projects or Products
- SOE
- SU Basic
- Adapt ( I think)

# Associated Issues and/or People
- D8CORE-2630

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
